v2/openapi/README.md

```diff
@@ -158,6 +162,15 @@ be replaced by a list of account IDs:
 Based on the response, Fireblocks platform will use the endpoints specific to each
 component to discover the specific capabilities for each component.
 
+Note that the `crosstransfers` component is used to define the capabilities for 
+on-ramp, off-ramp, and bridging without specifying a concrete account. 
+The actual transfers are executed using the **omnibus account**, which implies 
+the capabilities of that account. 
```

Comment:
That has changed, right? 
As agreed to start with only one use case, we might want to delete the other until we bring it back to the table. 
If it's part of MVP, please add more context and perhaps even an example for this capability. For someone who reads it for the first time, it's almost impossible to understand what the provider should implement given its business case.